app/views/provider_interface/organisation_settings/show.html.erb

```diff
@@ -4,16 +4,41 @@
   <div class="govuk-grid-column-two-thirds">
     <h1 class="govuk-heading-l"><%= t('page_titles.provider.organisation_settings') %></h1>
     <ul class="govuk-list govuk-list--spaced">
-      <% if current_provider_user.authorisation.can_manage_users_for_at_least_one_provider? %>
-        <li>
-          <%= govuk_link_to t('page_titles.provider.users'), provider_interface_provider_users_path %>
-        </li>
-      <% end %>
+      <% if FeatureFlag.active?(:account_and_org_settings_changes) %>
+        <% if @providers.count > 1 %>
+          <% @providers.each do |provider| %>
+            <h2 class="govuk-heading-m"><%= provider.name %></h2>
+            <li>
+              <%= govuk_link_to t('page_titles.provider.users'), provider_interface_provider_users_path %>
```

Comment:
Are these going to be updated to point to user lists for the specific provider in another card?

Reply:
Yes, we currently don't have an appropriate page to go to